src/entrypoint/graphql.ts

```diff
@@ -184,6 +193,7 @@ const resolvers = {
       return uid
     },
     getLevels: () => Levels,
+    getUserLimits: (_, __, { wallet }) => wallet.getUserLimits(),
```

Comment:
could we refactor and not have a `getLimits` and `getUserLimits`? not clear what would be the difference when just looking at the function name